s4e-web/cypress/integration/map/map-data-search.spec.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types="Cypress" />
+
+import { Login } from '../../page-objects/auth/auth-login.po';
+import { MapDataSearch } from '../../page-objects/map/map-data-search.po';
+
+before(function () {
+  cy.fixture('users/zkMember.json').as('zkMember');
+});
+
+describe.only('Map Data Search', () => {
+
+  context("Data search for not logged in user", () => {
+
+    before(() => {
+      cy.visit('/login');
+    });
+
+    it('shouldn\'t download data without authentication', () => {
+      Login
+        .goToMapWithoutLogin()
+      MapDataSearch
+        .goToSearchData()
+        .selectNthProductToSearch(0)
+        .searchData()
+        .selectNthDataToDownload(0)
+        .shouldReturnToLoginPage()
+    });
+  });
+
+  context("Data search for logged in user", () => {
+
+    beforeEach(function(){
+      cy.visit('/login');
+      Login
+        .loginAs(this.zkMember)
+    });
+
+    it("should download data", () => {
+      MapDataSearch
```

Comment:
After you had logged in, you are navigated to the map. In that case you should start 

```
Map
  .goToSentinelDataSearch() (inside change context to MapDataSearch)

Reply:
In the test there is no Map class.

Reply:
As you beginning with

```
Login
        .loginAs(this.zkMember)
        
```

Map page object is your last context